addon/components/breadcrumbs.js

```diff
@@ -1,4 +1,5 @@
 import Component from '@ember/component'
+import { action } from '@ember/object'
```

Comment:
This will bump our Ember source requirement, right?

Reply:
Correct. Totally forgot about this. Will close this for now.